rel/value_set.go

```diff
@@ -327,6 +327,48 @@ func (s *genericSet) Any() Value {
 	panic("Any(): empty set")
 }
 
+func (s *genericSet) AsString() (String, bool) {
+	if s.set.Count() == 0 {
```

Comment:
`!s.Bool()`

Reply:
Might be worth changing `Bool` to `IsTrue`. Raise an issue.

Reply:
is the purpose of `Bool` to show the `Bool` value of a set or to show if it is not empty? If it's the latter, might be better to change it to `NotEmpty`.

Reply:
All values have a "truthiness" to them, which determines how they affect logical operations. `0`, `()` and `{}` are all considered "false". Every other value is considered "true". The `Bool()` method expresses this, but `IsTrue` would be less confusing.